### ISSUE TITLE
Installation Issues

### ISSUE THREAD

**Issue by bckelley** [microsoft/terminal#14482]
### Windows Terminal version

Preview 1.16

### Windows build number

10.0.19045.0

### Steps to reproduce

Uninstall Windows Terminal Preview v1.16 or any software

### Actual Behavior

What's with leaving orphans when you uninstall software? Can we please have someone look into this and correct it before I completely switch to Linux

**Comment by zadjii-msft** [comment]
What _exactly_ is getting orphaned when you uninstall the Terminal/? Processes? Files? Reg keys? Hard to tell what the issue is when you haven't filled out the template 😜

**Comment by bckelley** [comment]
your template was filled out to the best of my ability with the information given ... if you wanted specifics you should say what information you wanted. In this case Files, I uninstall Terminal Preview v1.16 and those folders didn't delete from C:\Program Files\WindowsApps\ so I can only assume other programs will do the same thing and leave their respective folders behind as well.

**Comment by DHowett** [comment]
The application package contains a font that is registered automatically with the system. On occasion, the AppX service will purge unwanted system registrations; sometimes it cannot do that if the files are still in use (e.g. you visited a site or launched an application that tried to display text in the Cascadia Code font). It _should_ try again later.

`WindowsApps` is a system-managed folder that we don't actually have local control over. Sorry.

For what it's worth... if we could add every possible set of conditions to our bug template ("if you're reporting an installation issue, are you reporting a file orphaning issue? Okay, since you're reporting a file orphaning issue, which files are getting orphaned?") we would no longer be in need of gainful employment as we would have identified _all extant bugs._

In short: we can't ask for "unknown unknowns."
